journal/original/03-chapter-2.adoc

```diff
@@ -1084,21 +1084,21 @@ After it was a little abated,
 it froze so hard that we could not get over the river for a whole week.
 
 During our stay, we had disputes with several, and opened the principles of Truth to them.
-One major Taylor confessed to the Truth, and said.
-If the Quakers`' principles were such as we declared, he was a Quaker in judgment;
+One Major Taylor confessed to the Truth, and said,__
+If the Quakers`' principles were such as we declared, he was a Quaker in judgment;__
```

Comment:
Here I think you got it right in the `modernized` version, the `__` should be on the next line, next to "If", like so:

```ASCIIDOC
One Major Taylor confessed to the Truth, and said,
__If the Quakers`' principles were such as we declared, he was a Quaker in judgment;__
```